### PR TITLE
Creating Kits v1.1 and v2.2 that just pin "missing" plugins into the versions most suitable for the CLIs used

### PR DESCRIPTION
**Pull request by digeff**
Most of the PR is Just copying templates/default into templates/fixed-whitelist.

### CODE REVIEW

File: src/taco-kits/TacoKitMetadata.json [hunk 1]
```diff
@@ -56,6 +57,150 @@
...
+                },
+                "cordova-plugin-legacy-whitelist": {
+                    "version": "1.1.0"
+                }
+
+            }
+        },
+        "tacokit@2.1.0": {
+            "cordova-cli": "5.2.0",
+            "releaseNotesUri": "http://go.microsoft.com/fwlink/?LinkId=623759",
```
Comment (MSLaguana) [inline src/taco-kits/TacoKitMetadata.json]: This release notes link will need to be updated surely? To mention that these aren't validated plugins, but point-in-time?

**Comment by MSLaguana** [comment]
Were you also going to add a tacokit@3.0.0 for cordova 5.4.1, or are we waiting until cordova 6.0 or that?

**Comment by digeff** [comment]
We are waiting...

File: src/taco-kits/TacoKitMetadata.json (hunk 1 above)
Comment (digeff) [inline src/taco-kits/TacoKitMetadata.json]: Yes, we need to remember to do that!

Comment (meenakb) [inline src/taco-kits/TacoKitMetadata.json]: +1. Please open a bug for tracking it.

**Comment by meenakb** [comment]
:+1:

**Pull request closed without merging**